.changeset/brave-icons-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/onchainkit": patch
+---
+
+- **patch**: Remove address prop and fix sponsor component. By @abcrane123 #1114
```

Comment:
nit. try to be more verbose and explain this as someone with no context is reading this.